app/javascript/react/screens/App/Mappings/screens/MappingWizard/components/MappingWizardGeneralStep/__tests__/__snapshots__/MappingWizardGeneralStepReducer.test.js.snap

```diff
@@ -0,0 +1,359 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`fetching conversion hosts is pending 1`] = `
+Object {
+  "conversionHosts": Array [],
+  "editingMapping": null,
+  "errorFetchingConversionHosts": null,
+  "isFetchingConversionHosts": true,
+  "isRejectedConversionHosts": false,
+}
+`;
+
+exports[`fetching conversion hosts is rejected 1`] = `
+Object {
+  "conversionHosts": Array [],
+  "editingMapping": null,
+  "errorFetchingConversionHosts": "error",
+  "isFetchingConversionHosts": false,
+  "isRejectedConversionHosts": true,
+}
+`;
+
+exports[`fetching conversion hosts is successful and there is data 1`] = `
```

Comment:
This snapshot is a bit unwieldy, can we change the test to keep this more readable?